src/AccessorPairAsserter.php

```diff
@@ -4,19 +4,23 @@
 namespace DigitalRevolution\AccessorPairConstraint;
 
 use DigitalRevolution\AccessorPairConstraint\Constraint\AccessorPairConstraint;
+use DigitalRevolution\AccessorPairConstraint\Constraint\ConstraintConfig;
 use PHPUnit\Framework\Assert;
 
 trait AccessorPairAsserter
 {
     /**
-     * @param string $object               The fully qualified name of the class that should be tested
-     * @param bool   $testPropertyDefaults When true, the getter methods are called on an empty instance of the test object.
-     *                                     This makes sure that all the properties have the correct default type,
-     *                                     conforming the getter return typehint.
-     * @param string $message              Custom PHPUnit error message in case of constraint failure
+     * @param string           $object  The fully qualified name of the class that should be tested
+     * @param ConstraintConfig $config  Configuration of the constraint.
+     *                                  By default the getter/setter pairs and constructor/getter pairs will be tested
+     * @param string           $message Custom PHPUnit error message in case of constraint failure
      */
-    public static function assertAccessorPairs(string $object, bool $testPropertyDefaults = false, string $message = '')
+    public static function assertAccessorPairs(string $object, ConstraintConfig $config = null, string $message = '')
```

Comment:
As the implementation of the "assertAccossorPairs" function is now changed, can you add an upgrade guide/breaking changes section somewhere?